### DISCUSSION TITLE
StableSwarmUI on Intel Mac?

### DISCUSSION THREAD

**Discussion by raffetazarius** [Stability-AI/StableSwarmUI#207] [Q&A]
I know it says on the main page that StableSwarmUI only supports Apple Silicon, but figured I'd ask anyway if anyone had managed to get it running on an Intel Mac?   If so, how did you get it working?

I'm on a Mac Pro 2019 with a Radeon W5700X 16GB MPX GPU and a Radeon RX 6900 XT 16 GB Reference GPU.

**Comment by mcmonkey4eva** [comment]
Swarm itself is hypothetically compatible with intel mac, the problem is the backend (ie comfy)

It's _possible_ that it will work if you follow the Linux AMD instructions for Comfy: <https://github.com/comfyanonymous/ComfyUI#amd-gpus-linux-only>
If not, you'll have better luck asking directly in the comfy github than here. If/when you get an answer that applies to comfy, it automatically applies to swarm as well - if you find one, please do share it back here though, I'd love to set Swarm up to Just Work :tm: automatically in that env.
